src/utils/imsc1-ttml-parser.ts

```diff
@@ -182,9 +186,20 @@ function getTtmlStyles(region, style): { [style: string]: string } {
     // 'direction',
     // 'writingMode'
   ];
+
+  const regionStyleName = region.hasAttribute('style')
```

Comment:
I've been trying to generate some test content for imsc+hls so that it can be available for testing in the long term. One of the test files I got ends up not having a `region` at all, which causes this to fail. Probably should add a guard for it.
```suggestion
  const regionStyleName = region && region.hasAttribute('style')
```

Reply:
Also, with that, I think the default region styles aren't quite correct, but I think that shouldn't necessarily be part of this PR.

Reply:
If you have any test content that you're able to share, that would be helpful.

Reply:
Hi @gkatsev 
[1.m4s.zip](https://github.com/video-dev/hls.js/files/8040770/1.m4s.zip)
This segment was used for testing.
When you see the file structure, you will understand the addition of the style to the region. 
I don't have test streams right now

Reply:
Thanks. I've been trying to generate an mp4 file like that, but all I can find will generate it with the xml documents being in the same mdat instead of across multiple mdats (which I don't think it correct).

Reply:
I generated a test source to test some IMSC in HLS here https://test-streams.mux.dev/tos_ismc/main.m3u8